File: asp.netcore/User_Registration_Demo.Domain/DTO/SubscriptionDto.cs
using System.ComponentModel.DataAnnotations;

namespace User_Registration_Demo.Domain.DTO
{
    public class SubscriptionDto
    {
        public int? Id { get; set; }
        public string? PackageType { get; set; }
        public decimal PackageCost { get; set; }
        public decimal PackageDiscount { get; set; }
        public List<SubscriptionFeaturesDto>? SubscriptionFeaturesList { get; set; }
    }
}

File: angular/src/app/main/Models/SubscriptionDto.ts
export interface SubscriptionDto {
    id?: number;
    packageType?: string;
    packageCost?: number;
    packageDiscount?: number;
    subscriptionFeaturesList?: SubscriptionFeaturesDto[];
    numberOfBranches?: number;
}

export interface SubscriptionFeaturesDto {
    featureName: string;
}